libs/ledger/src/chain/block_coordinator.cpp

```diff
@@ -517,6 +519,8 @@ BlockCoordinator::State BlockCoordinator::OnSynchronised(State current, State pr
   FETCH_LOG_INFO(LOGGING_NAME, "Minting new block! Number: ", next_block_->block_number,
                  " beacon: ", next_block_->block_entropy.EntropyAsU64());
 
+  start_block_packing_ = Clock::now();
```

Comment:
Feels like this would be better to use the histogram data that already exists in telemetry